main.go

```diff
@@ -185,8 +184,6 @@ func main() {
 	root.AddCommand(remoterun.RemoteRun(ctx, k8sLogger))
 	root.AddCommand(test.Test(ctx, ioController, k8sLogger, at))
 
-	// deprecated
-	root.AddCommand(stack.Stack(ctx, at, insights, ioController))
 	root.AddCommand(pipeline.Pipeline(ctx))
```

Comment:
deprecated is still applying for pipeline?

Reply:
No, I removed the `//deprecated` to reflect it. We should probably needed to put it on the side instead of the top of the function.